etna/core/wagtail_hooks.py

```diff
@@ -19,6 +19,11 @@ def editor_js():
     )
 
 
+@hooks.register("insert_global_admin_js")
+def global_admin_js():
+    return "<script>window.chooserUrls = {'pageChooser': '/admin/choose-page/','externalLinkChooser': '/admin/choose-external-link/','emailLinkChooser': '/admin/choose-email-link/','phoneLinkChooser': '/admin/choose-phone-link/','anchorLinkChooser': '/admin/choose-anchor-link/',}; </script> <script src='/static/wagtailadmin/js/modal-workflow.js?v=4a9c2a53'></script>"
+
```

Comment:
Hi Allan, I copied this directly from the Wagtail internals as it was missing from certain views 😃

Reply:
As to will it ever change, I'm not sure. Because I grabbed it from Wagtail, I should be able to just grab it again if needed (I'm guessing this number is the version number they use or something?) 🙂

Reply:
>  I copied this directly from the Wagtail internals as it was missing from certain views

If its internal I hope that value does not change?

While the solution works, it does seem like a hardcoded value and I was wondering if this would be specific for a Wagtail version - as such will it ever change? If so then this value would need updating.

Do you mind recording in the ticket the steps taken to get '</script> <script src='/static/wagtailadmin/js/modal-workflow.js?v=4a9c2a53'></script>'  for the benefit of any developer ?

I am approving it as of now.